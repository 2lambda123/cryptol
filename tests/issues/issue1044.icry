:set prover=sbv-z3
:safe \(x:Z 7) -> x * (recip x) == 1

:set prover=w4-z3
:safe \(x:Z 7) -> x * (recip x) == 1
