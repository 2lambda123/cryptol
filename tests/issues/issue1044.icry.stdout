Loading module Cryptol
Counterexample
(\(x : Z 7) -> x * (recip x) == 1) 0 ~> ERROR
division by 0
-- Backtrace --
Cryptol::recip called at issue1044.icry:2:24--2:29
(Cryptol::*) called at issue1044.icry:2:19--2:37
(Cryptol::==) called at issue1044.icry:2:19--2:37
<interactive>::it called at issue1044.icry:2:7--2:37
Counterexample
(\(x : Z 7) -> x * (recip x) == 1) 0 ~> ERROR
division by 0
-- Backtrace --
Cryptol::recip called at issue1044.icry:5:24--5:29
(Cryptol::*) called at issue1044.icry:5:19--5:37
(Cryptol::==) called at issue1044.icry:5:19--5:37
<interactive>::it called at issue1044.icry:5:7--5:37
